### ISSUE TITLE
requiresMainQueueSetup error with RN 0.51

### ISSUE THREAD

**Issue by Ayc0** [tanguyantoine/react-native-music-control#125]
# Description
Got the error:
```
Module MusicControlManager requires main queue setup since it overrides
`constantsToExport` but doesn't implement `requiresMainQueueSetup`.
In a future release React Native will default to initializing all native modules
on a background thread unless explicitly opted-out of.
```

1. Platform ?
   - [x] iOS
   - [ ] Android
   
   
2. Device
  - [x] Simulator
  - [x] Real device

3. RN version
0.51.0

**Comment by Ayc0** [comment]
How to fix it:
https://github.com/wix/react-native-navigation/pull/1983/files#diff-b8c48202d12fa78c4c377e5a3225c298

**Comment by tanguyantoine** [comment]
fix by #126
